daisy_workflows/image_build/windows_for_containers/container_image_install_ce.ps1

```diff
@@ -151,6 +151,19 @@ function Run-FirstBootSteps {
   Write-Host 'Running docker install script'
   $dockerwait = Start-Process -PassThru -FilePath "powershell.exe" -ArgumentList "$env:TEMP\install-docker-ce.ps1"
   Wait-Process -InputObject $dockerwait
+
+  Write-Host 'Copying Docker license information'
+  mkdir C:\Windows\System32\Docker
```

Comment:
Just so this doesn't get committed. James and I chatted about a set of changes to this PR.